RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -25,6 +25,7 @@
             <BitmapImage x:Key="UndoImage" UriSource="../../Resources/arrow-circle-left.png" />
             <BitmapImage x:Key="PrintImage" UriSource="../../Resources/printer.png" />
             <BitmapImage x:Key="AddTestModuleImage" UriSource="../../Resources/Custom/PNG/ListOfTests.png" />
+            <BitmapImage x:Key="AddTestModuleWithStubsImage" UriSource="../../Resources/Custom/PNG/ListOfTests.png" />
```

Comment:
Why load the same resource twice?

Reply:
It's because I have a negative level of WPF knowledge. Will be fixed.